### DISCUSSION TITLE
idea for procedural fragments in the beta

### DISCUSSION THREAD

**Discussion by splofie** [studio-minus/ppg-bugs-and-requests#2504] [Ideas]
Could you make it possible for the procedural fragments to be generated by explosions? I just thought the effect would go really well with it.

**Comment by ghost** [comment]
It is? Or do you mean the grenade particles or what

**Reply by dinevolence** [reply]
He probably meant procedural dust particles that would be launched or something.

**Reply by splofie** [reply]
I should've specified, i meant if an explosion strong enough could blow their head up.
example: A dynamite close enough to a humans head would blow it up/ turn it into the fragments instead of just decapitating.

**Reply by dinevolence** [reply]
> I should've specified, i meant if an explosion strong enough could blow their head up. example: A dynamite close enough to a humans head would blow it up/ turn it into the fragments instead of just decapitating.

Isn't that already the case?

**Reply by splofie** [reply]
> > I should've specified, i meant if an explosion strong enough could blow their head up. example: A dynamite close enough to a humans head would blow it up/ turn it into the fragments instead of just decapitating.
> 
> 
> 
> Isn't that already the case?

Nope.

**Comment by mestiez** [comment]
agree
